docs/src/operate/c8y/c8y_service_monitoring.md

```diff
@@ -11,59 +11,62 @@ or on the `child` device can be monitored from the **Cumulocity IoT** by sending
 
 ## Send the health status of a service to health topic.
 
-The table below lists the MQTT topics to which the health status message should be sent, and the
-health status message format for both the `thin-edge` and for the `child` device.
+A health status message can be published for any service on a `status/health` channel. The health message has to contain
+at least the `status` of the service.
 
-|Device-type|Thin-edge-Health-status-mqtt-topic|Health-status-message|
-|------|------------------------|---------------------|
-|Thin-edge-device|`tedge/health/<service-name>`|`{"status":"status of the service","type":"type of service"}`|
-|Child-device|`tedge/health/<child-device-id>/<service-name>`|`{"status":"status of the service","type":"type of service"}`|
+**TODO: Link to the Health Status Message reference!**
```

Comment:
For now I removed it, because technically it was a TODO for introducing more documentation. I written it because I wasn't sure if updating was really everything that should be done.

IMO we have some unnecessary duplication going on in the documentation, e.g. between _Raising Alarms_, _Data model_, and _Thin Edge JSON_ pages, where on _Raising Alarms_ page `text` field is described as optional, but in _Thin Edge JSON_ page in _Alarms_ section reference table, it isn't.

I think that instead of describing all the fields and their properties in a _Getting started_ chapter isn't really necessary, and it's better to have a _Reference_ section, where all the message types are described in detail, more formally, and then have a _Getting Started_ section just pick the most important information from the reference, and describe it with less formal language and with more examples.

Not to say that it's all bad, but I feel the current structure of the documentation makes it easy to miss things. I'd love to discuss this when you're available.

Reply:
Yes the reworking of the telemetry data pages is planned after the 0.13.0 as it will require a bit more thought on how best to display the information in a concise manner.